core/trino-main/src/test/java/io/trino/execution/scheduler/TestArbitraryDistributionSplitAssigner.java

```diff
@@ -460,10 +460,10 @@ public void testAdaptiveTaskSizing()
                 partitionedSources,
                 ImmutableSet.of(),
                 1,
-                1.2,
-                1,
-                4,
-                STANDARD_SPLIT_SIZE_IN_BYTES,
+                1.3,
+                100,
+                400,
+                100,
```

Comment:
can you please leave original test parameters intact and introduce separate testcase focused on rounding; so test of standard flow does not use super artificial configuration params.